### PR TITLE
OSS push from bbpcode repository

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMake/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Makefile
```
Comment (eile) [inline CMake/.gitignore]: ?

Comment (tribal-tec) [inline CMake/.gitignore]: ? The one from Buildyard. That's not checked in imo

Comment (eile) [inline CMake/.gitignore]: Me confused. BY does not put a Makefile in CMake/.

**Comment by eile** [comment]
+1

File: CMake/.gitignore (hunk 1 above)
Comment (tribal-tec) [inline CMake/.gitignore]: Ah, CMake/.gitignore. Dunno where this comes from, I remove it...

**Pull request merged**